src/components/Chart/ScatterPlots.js

```diff
@@ -318,45 +326,46 @@ const ScatterPlot = ({
         y={cy + (top ? -yOffset : yOffset)}
         contextWidth={width}
       >
-        {hover.map(({ value }, i) => (
-          <ContextBoxValue
-            key={`${value.datum[label]}${i}`}
-            label={value.datum[label]}
-          >
-            {intersperse(
-              []
-                .concat(
-                  value.datum[detail] &&
-                    value.datum[detail]
-                      .split('\n')
-                      .map((d, i) => (
-                        <Fragment key={`d${i}`}>{subsup(d)}</Fragment>
-                      ))
-                )
-                .concat([
-                  yShowValue && (
-                    <Fragment key='y'>
-                      {yFormat(value.y)} {subsup(yUnit)}
-                    </Fragment>
-                  ),
-                  xShowValue && (
-                    <Fragment key='x'>
-                      {xFormat(value.x)} {subsup(xUnit)}
-                    </Fragment>
-                  ),
-                  sizeShowValue && (
-                    <Fragment key='size'>
-                      {sizeFormat(value.size)} {subsup(sizeUnit)}
-                    </Fragment>
-                  )
-                ])
-                .filter(Boolean),
-              (item, index) => (
-                <br key={`br${index}`} />
-              )
-            )}
-          </ContextBoxValue>
-        ))}
+        {hover.map(({ value }, i) => {
+          const formattedValues = {
+            y: yFormat(value.y),
+            x: xFormat(value.x),
+            size: sizeFormat(value.size)
+          }
+          const contextT = text => {
+            const replacements = getReplacementKeys(text).reduce(
+              (acc, replacementKey) => {
+                acc[replacementKey] =
+                  acc[replacementKey] ||
+                  value[replacementKey] ||
+                  value.datum[replacementKey] ||
+                  props[replacementKey]
```

Comment:
what are the uses cases for exposing props? I'd prefer a well defined set of replacement ala:
```
{
  value: 10000,
  formattedValue: '10'000',
  ...value.datum // all data column values
}
```

Reply:
we also have the `unit` (or `xUnit`and `yUnit`) which could be useful. But yes that's about it.

Reply:
+ add replacement info in docs